### PR TITLE
add avatar change resgiter user

### PR DESCRIPTION
**Pull request by minhld1709**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -162,5 +162,15 @@ $gray-medium-light: #eaeaea;
 
 header img {
   width: 40px;
-  height: 23px;
+  height: 24px;
+}
+
+.header_user{
```
Comment (phonghenry14) [inline app/assets/stylesheets/custom.css.scss]: ???

File: Gemfile [hunk 2]
```diff
@@ -6,6 +6,7 @@ gem 'bootstrap-sass',       '3.2.0.0'
 gem "factory_girl_rails"
 gem "faker"
 gem "cancancan"
+gem "carrierwave"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.1'
```
Comment (daothanhcam) [inline Gemfile]: change all ' to "

File: app/views/admin/users/edit.html.erb [hunk 3]
```diff
@@ -24,11 +24,15 @@
...
       <%= f.password_field :password, autocomplete: "off", class: "form-control" %>
-  </div>
+    </div>
     <div class="col-md-12 field">
       <%= f.label t "admin.user.create.pw_cf" %>
       <%= f.password_field :password_confirmation, autocomplete: "off", class: "form-control" %>
     </div>
+    <div class="col-md-12 field">
+      <%= f.label t "admin.user.create.choice_avt" %>
+      <%= f.file_field :image  %>
```
Comment (phonghenry14) [inline app/views/admin/users/edit.html.erb]: thừa space

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -162,5 +162,15 @@ $gray-medium-light: #eaeaea;
...
+
```
Comment (daothanhcam) [inline app/assets/stylesheets/custom.css.scss]: remove this line.

File: app/controllers/admin/users_controller.rb [hunk 5]
```diff
@@ -7,6 +7,10 @@ def new
 
   def create
     @user = User.new user_params
+    @user.birthday = Faker::Time.backward(14, :evening)
+    pw = "12345678";
+    @user.password = pw;
+    @user.password_confirmation = pw;
```
Comment (daothanhcam) [inline app/controllers/admin/users_controller.rb]: remove semicolon

File: app/controllers/admin/users_controller.rb [hunk 6]
```diff
@@ -31,6 +35,6 @@ def update
 
   private
   def user_params
-    params.require(:user).permit :name, :email, :birthday, :password, :password_confirmation
+    params.require(:user).permit :name, :email, :birthday, :password, :password_confirmation, :image
```
Comment (daothanhcam) [inline app/controllers/admin/users_controller.rb]: This line too long.

File: app/views/admin/users/edit.html.erb (hunk 3 above)
Comment (daothanhcam) [inline app/views/admin/users/edit.html.erb]: Remove blank space.

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -19,14 +19,16 @@
...
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown"><%= current_user.name %>
-                <b class="caret"></b></a>
-              <ul class="dropdown-menu">
-                <li><%= link_to t("layouts.profile"), "#" %></li>
-                <li><%= link_to t("layouts.setting"), "#" %></li>
-                <li><%= link_to t("layouts.log_out"), sign_out_path %></li>
-              </ul>
+            <li id="fat-menu" class="dropdown header_user">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                  <%= image_tag(current_user.image_url.to_s) %>
```
Comment (daothanhcam) [inline app/views/layouts/_header.html.erb]: Try remove ()

**Pull request merged**